cmd/krew/cmd/info.go

```diff
@@ -25,38 +25,41 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	"sigs.k8s.io/krew/internal/info"
+	"sigs.k8s.io/krew/internal/index/indexscanner"
 	"sigs.k8s.io/krew/internal/installation"
+	"sigs.k8s.io/krew/internal/pathutil"
 	"sigs.k8s.io/krew/pkg/index"
 )
 
 // infoCmd represents the info command
 var infoCmd = &cobra.Command{
 	Use:   "info",
-	Short: "Show information about a kubectl plugin",
-	Long: `Show information about a kubectl plugin.
+	Short: "Show information about an available plugin",
+	Long: `Show detailed information about an available plugin.
 
-This command can be used to print information such as its download URL, last
-available version, platform availability and the caveats.
+Examples:
+  kubectl krew info PLUGIN
+  kubectl krew info INDEX/PLUGIN`,
```

Comment:
cobra.Command has an `Examples` section. This should go in there.

Reply:
IIRC it’s a single Example field. So we need to figure out what indentation does the template use so we can have multiple lines.

Reply:
Ah, just saw that we don't use it in krew. So nevermind..

Reply:
We use it in _certain_ cmds that have a single example. But `install`, `search` have rather complex examples, so they're in LongDescription.